src/social-links.json

```diff
@@ -1,4 +1,13 @@
 {
     "social_media_name": "link",
-    "linkedin": "https://www.linkedin.com/in/"
-}
\ No newline at end of file
+    "linkedin": "https://www.linkedin.com/in/",
+    "miscellaneous": {
```

Comment:
We don't need to add another dictionary named miscellaneous, it was just for our reference, you can directly add the handles.